### DISCUSSION TITLE
Can I have limited use of the ICONS drawn for various applications on the home page of Dracula theme

### DISCUSSION THREAD

**Discussion by JavieDeng** [dracula/dracula-theme#837] [Q&A] [answered]
Can I have limited use of the ICONS drawn for various applications on the home page of Dracula theme.I think those ICONS are beautiful. Can I use them elsewhere?

**Comment by luxonauta** [comment, marked as answer]
Unfortunately not; these icons were licensed by purchasing through the FlatIcon platform. 😅

✅ Accordingly, we have the non-exclusive, non-transferable, non-sublicensable right to use the licensed icons.

❌ We cannot sublicense, sell or rent any (or a modified version).
